L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -160,9 +179,20 @@ std::vector<std::pair<GlobalPoint, double>> HGCalHistoSeedingImpl::computeMaxSee
         if (isMax) {
           float ROverZ_seed = kROverZMin_ + (bin_R + 0.5) * (kROverZMax_ - kROverZMin_) / nBinsRHisto_;
           float phi_seed = -M_PI + (bin_phi + 0.5) * 2 * M_PI / nBinsPhiHisto_;
-          float x_seed = ROverZ_seed * cos(phi_seed);
-          float y_seed = ROverZ_seed * sin(phi_seed);
-          seedPositionsEnergy.emplace_back(GlobalPoint(x_seed, y_seed, z_side), MIPT_seed);
+
+          float x_seed = 0;
+          float y_seed = 0;
+
+	  if ( seedingPosition_ == BinCentre ){
```

Comment:
Here (and in similar code portions in other methods), there are mixed indentations to be fixed